File: Easy/Python/1909. Remove One Element to Make the Array Strictly Increasing.py
class Solution(object):
    def canBeIncreasing(self, nums):
        count = 0

        for i in range(1, len(nums)):
            if nums[i] <= nums[i - 1]:
                count += 1
                if 1 < count:
                    return False
                if 1 < i and nums[i] <= nums[i - 2]:
                    nums[i] = nums[i - 1]
        
        return True

File: Easy/Java/1909. Remove One Element to Make the Array Strictly Increasing.java
class Solution {
    public boolean canBeIncreasing(int[] nums) {
        int count = 0;

        for(int i = 1; i < nums.length; i++){
            if(nums[i] <= nums[i - 1]){
                count++;
                if(1 < count)
                    return false;
                if(1 < i && nums[i] <= nums[i - 2])
                    nums[i] = nums[i - 1];
            }
        }

        return true;
    }
}
